src/app/components/pool/details/pool-cost-card/pool-cost-card.html

```diff
@@ -7,7 +7,7 @@
             [attr.aria-label]="label">
         </canvas>
     </div>
-    <div class="total">
+    <div class="total" aria-live="polite">
```

Comment:
Can we just remove this warning instead? April 2019 is a long time ago

Reply:
Good call. Removing section.